gyp/pylib/gyp/MSVSNew.py

```diff
@@ -191,7 +191,8 @@ def __init__(self, path, version, entries=None, variants=None,
       self.variants = variants[:]
     else:
       # Use default
```

Comment:
This is a little beyond my current GYP depth, so please forgive me if this is a dumb question: wouldn't it be better to have GYP pass in the correct Win32+ARM64 variants instead of emitting both even when not cross-compiling?

Reply:
Yes, that would be better, I will look at it.

Reply:
Looked at the variants and removed ARM64 from default. The platform variants is passed properly during configuration phase.